packages/modern-plugin-rspress/src/features/lanchDoc.ts

```diff
@@ -129,6 +129,7 @@ export async function launchDoc({
         locales,
       },
       markdown: {
+        mdxRs: false,
```

Comment:
can remove this line ? we can unable it  in plugin which unsupport mdxRs

Reply:
Currently we disabled it in all the places. Once the mdx-rs support relavant plugins, we will store the config.